portfolio/src/main/webapp/script.js

```diff
@@ -48,3 +48,33 @@ function addPhoto() {
   imageContainer.innerHTML = '';
   imageContainer.appendChild(imgElement);
 }
+
+function getCommentSectionFromServer() {
+  fetch('/comment-section').then(response => response.json()).then((comments) => {
+    const commSection = document.getElementById('comment-list');
+    // Build the comment section.
+    comments.forEach((comm) => {
+      commSection.appendChild(createIndividualComment(comm));
+    });
+  });
+}
+
+// Format each particular comment to show up on page.
+function createIndividualComment(comment) {
+  const liElement = document.createElement('li');
+  // This container is used for showing the username.
+  const usernameElement = document.createElement('p');
+  // This container is used for showing the text of the comment.
+  const commentElement = document.createElement('p');
+
+  usernameElement.innerText = comment.username + ' wrote:';
```

Comment:
Optional: You might want to consider rendering the text and username differently, for example, how github markdown renders this:

`alexcepoi` wrote: `This is my comment.`

Reply:
I am not sure what do you mean by this, like to be different types of elements?

Reply:
Just that you could tweak syling a bit.